include/SDL2/SDL_config.h

```diff
@@ -51,6 +51,7 @@
 #include "SDL_config_unix.h"
 #else
 /* This is a minimal configuration just to get SDL running on new platforms. */
+#warning SDL_config.h might be incomplete, good luck
```

Comment:
Is `#warning` directive universally supported? A `#pragma message("...")` may also be considered here.

Reply:
I believe `#warning` is more widely supported than `#pragma message()`